kolibri/content/utils/transfer.py

```diff
@@ -45,8 +45,11 @@ def __init__(self, source, dest, block_size=2097152, remove_existing_temp_file=T
         self.finalized = False
         self.closed = False
 
-        signal.signal(signal.SIGINT, self._kill_gracefully)
-        signal.signal(signal.SIGTERM, self._kill_gracefully)
+        # TODO (aron): Instead of using signals, have bbq/iceqube add
+        # hooks that the app calls every so often to determine whether it
+        # should shut down or not.
```

Comment:
Ah okay, I see, for the graceful part

An interesting stale bug: http://bugs.python.org/issue14073